Podfile

```
@@ -0,0 +1,20 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'zmoney' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for zmoney
+  pod 'SwiftLint'
```

Чтобы добиться стабильности и не бояться, что новая версия зависимости сломает поведение, лучше явно указывать ревизии используемых зависимостей.
```suggestion
  pod 'SwiftLint', '~> 0.43.0'
```